hathor/manager.py

```diff
@@ -914,6 +914,8 @@ def push_tx(self, tx: Transaction, allow_non_standard_script: bool = False,
         if not tx_from_lib.is_standard(max_output_script_size, not allow_non_standard_script):
             raise NonStandardTxError('Transaction is non standard.')
 
+        tx.update_reward_lock_metadata()
```

Comment:
Why don't we remove the verify and capture the exception from the propagate_tx?

Reply:
Sure, I missed this one. Done in https://github.com/HathorNetwork/hathor-core/pull/1008/commits/5dc99fa98c4cb3ed23aa1ac66d03862d964eb5b2

Reply:
Thanks!